### ISSUE TITLE
In high contrast themes, keyboard focus indicator is not visible on the values of 'Action' column header popup.

### ISSUE THREAD

**Issue by msftedad** [ag-grid/ag-grid#7054]
**Repro steps:**

1. Launch the URL: [JavaScript Data Grid: Keyboard Interaction (ag-grid.com)](https://www.ag-grid.com/javascript-data-grid/keyboard-navigation/)
2. New screen starts appearing.
3. Navigate to 'country' under example in "Floating filters'' Section and reach hamburger menu on sports using 7 tab keys from example.
4. Invoke hamburger menu and turn on high contrast themes.
5. Verify if keyboard focus indicator is visible or not on column header pop-up.

**Current behavior**
In high contrast (Aquatic, Desert, Dusk & Night sky) themes, keyboard focus indicator is not visible on the values of 'Action' column header popup present on the "Vertical Splitter Layout" screen.

**Expected behavior**
In high contrast (Aquatic, Desert, Dusk & Night sky) themes, keyboard focus indicator is not visible on the values of 'Action' column header popup present on the "Vertical Splitter Layout" screen.

**Please tell us about your environment:**

* **AG Grid version:** X.X.X
27.3.0

* **Browser:**
Firefox Version 112.0.2 (64-bit)

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
ES5

****Attachments:**
**Normal mode:****
![MAS4 3 1_Normal_Data grid](https://github.com/ag-grid/ag-grid/assets/93735775/63cb412a-535e-4c9a-9180-84cadfb4a668)

**Dusk mode:**
<img width="960" alt="Dusk mode" src="https://github.com/ag-grid/ag-grid/assets/93735775/f81746b1-ccae-4025-aa9e-473c2e944cc6">

**Night sky mode:**
<img width="960" alt="Night sky mode" src="https://github.com/ag-grid/ag-grid/assets/93735775/fe11cb10-130b-47ff-a879-007470ad4f16">

**Desert mode:**
![MAS4 3 1_Desert_Data grid](https://github.com/ag-grid/ag-grid/assets/93735775/62a5326e-34bd-4172-8e6f-9e7d7813c360)

**Aquatic mode:**
![MAS4 3 1_Aquatic_Data grid](https://github.com/ag-grid/ag-grid/assets/93735775/ce020eef-aa22-44ff-ad4d-f85d64ec4b93)

**Comment by AG-Zoheil** [comment]
Hello @msftedad,

Thank you for the suggestion.

Currently there's no way to implement this behavior with AG Grid. However, we agree this would be a valuable addition to the product and we have added a feature request for this.

We have added this requirement to our backlog and we are tracking it with the following reference and description:
**AG-6967 [Accessibility] Allow the column/context menu items to display focused status when using a high-contrast theme**

We have no timeline for this at the moment, but this can still be picked up by our team when they're making modifications to this functional area of AG Grid. Please use the functionality currently provided by AG Grid until this is delivered at a later point.

See whether this item will be in the next release by checking the NEXT RELEASE checkbox on the product pipeline page:
https://www.ag-grid.com/ag-grid-pipeline/

The best way to track this is to sign up for AG Grid new release notifications using [the instructions here](https://ag-grid.zendesk.com/hc/en-us/articles/360002648058-How-do-I-sign-up-for-ag-Grid-release-notifications-). This way you'll know as soon as a new version is out and you can check whether this specific item was implemented on [the changelog page](https://www.ag-grid.com/ag-grid-changelog/).

Thanks again for bringing this up with us.

Kind regards,
Zoheil
